test-fixtures/src/bin/object_store.rs

```diff
@@ -0,0 +1,23 @@
+//! A guest program to test that object store works properly.
+
+use fastly::ObjectStore;
+
+fn main() {
+    let store_one = ObjectStore::open("store_one").unwrap().unwrap();
```

Comment:
Can we add a negative version of this test, too?

Reply:
Added in 4dc177e